Source/MLX/Documentation.docc/Organization/arithmetic.md

```diff
@@ -98,10 +98,26 @@ Note: the `-` and `/` operators are not able to be linked here.
 - ``MLXArray/.>=(_:_:)-6zxj9``
 - ``MLXArray/.&&(_:_:)``
 - ``MLXArray/.||(_:_:)``
+- ``MLXArray/&(_:_:)-9in7a``
+- ``MLXArray/&(_:_:)-8js41``
+- ``MLXArray/&(_:_:)-7iu4h``
+- ``MLXArray/|(_:_:)-2upd6``
+- ``MLXArray/|(_:_:)-35lfz``
+- ``MLXArray/|(_:_:)-7c6lr``
+- ``MLXArray/^(_:_:)-8hj69``
+- ``MLXArray/^(_:_:)-2oktd``
+- ``MLXArray/^(_:_:)-dik7``
+- ``MLXArray/<<(_:_:)-1dfss``
+- ``MLXArray/<<(_:_:)-1dfss``
+- ``MLXArray/<<(_:_:)-7hmgt``
+- ``MLXArray/>>(_:_:)-zpp3``
+- ``MLXArray/>>(_:_:)-4z891``
+- ``MLXArray/>>(_:_:)-89b4j``
```

Comment:
Picking up the bitwise ops from an earlier release (they were missing from mlx-c)